### ISSUE TITLE
Update to Python3

### ISSUE THREAD

**Issue by wimpers** [openvstorage/home#42]
Currently we use python  2.7.  This ticket is logged for the to update to Python3.

**Comment by khenderick** [comment]
It might be a good idea to make it python-version agnostic.

**Comment by toolslive** [comment]
No. Supporting python2 & python3 in the same code base is ~~a~~ ~~nightmare~~  too difficult.
